### ISSUE TITLE
Add another file on selection tree generation to indicate 

### ISSUE THREAD

**Issue by carmenfan** [3drepo/3drepobouncer#144]
Add a new file: `modelProperties.json `

which all it has is a list of unique ID of hidden nodes.

another feature for https://github.com/3drepo/3drepo.io/issues/235

**Comment by carmenfan** [comment]
@timscully1983  @camden-kid 

The tree generation algorithm now generates 2 json files:
![image](https://cloud.githubusercontent.com/assets/11945337/17856219/516899ec-6874-11e6-90af-a493642155d5.png)


If there are no hidden nodes in this scene, `modelProperties.json` does not exist.

with `modelProperties.json` look something like this: 

````javascript
{
"hiddenNodes":[
"01f82d6a-265f-4b96-8dae-ff98630001ef",
"df3248e9-14fe-4401-9204-59231799f58f",
"9fe348a2-8eed -4010-9d6b-94c29367f356",
"75de09a5-d0ee-45c7-aa4e-24174d8125dc",
"4d570cf8-df05-4bef-b2a9-84223d3fad13",
"0a861d57-dd99-41df-883a-ee3437cd5dfc",
"cec00556-8d21-437 b-a56b-ce46f698c8ec",
"be0b3fea-e074-425f-a553-cf6845ea1c68",
"554d1cb8-e669-420b-a2df-f718a86da164",
"74d97764-4f6b-4c90-8fc3-395544b4644f"
]
}
````

**Comment by carmenfan** [comment]
merged in and tagged as 1.3.6
